todoman/interactive.py

```diff
@@ -167,7 +156,9 @@ def __init__(self, parent, callback, database):
         is responsible for passing information to the underlying Page.
         It is usually a method of the underlying Page.
 
-        (ItemListPage, Main, function, Database) -> None
+        :param parent: Main
```

Comment:
That's not the right format, it's `:param TYPE NAME`, for example `:param int amount`.